hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -246,6 +234,8 @@
   private AtomicLong compactedCellsSize = new AtomicLong();
   private AtomicLong majorCompactedCellsSize = new AtomicLong();
 
+  private StoreContext storeContext;
```

Comment:
Should this be final? I wouldn't expect you would need to change StoreContext (the reference) after it has been initialized

Reply:
you're right, it could be final, will do it in next commit